src/cdk/table/table.spec.ts

```diff
@@ -109,10 +109,10 @@ describe('CdkTable', () => {
   });
 
   it('should render cells even if row data is falsy', () => {
-    const evenSimplerCdkTableAppFixture = TestBed.createComponent(EvenSimplerCdkTableApp);
+    const booleanRowCdkTableAppFixture = TestBed.createComponent(EvenSimplerCdkTableApp);
```

Comment:
Name of the component too?
`EvenSimplerCdkTableApp` -> `TableWithBooleanDataSource`

Reply:
d'oh yes - my brain was thinking they were all renaming